docs/guides/docs/quickstart/building-a-frontend.mdx

```diff
@@ -28,12 +28,12 @@ Once you've installed the wallet, take the address of your wallet and use it to
 
 To split our project's contract from frontend code, let's create a new folder `frontend` where we'll initialize our frontend project.
 
-In the terminal, go back up one directory and initialize a react project using [`Create React App`](https://create-react-app.dev/).
+In the terminal, go back up one directory into the `fuel-project` folder and initialize a react project using [`Create React App`](https://create-react-app.dev/) and TypeScript.
 
-```console
-$ cd ..
-$ npx create-react-app frontend --template typescript
-Success! Created frontend at Fuel/fuel-project/frontend
+<span id="create-react-app"></span>
```

Comment:
I mentioned this in another comment also but, would be good to use a custom component like `<TestAction>` and receive more info directly from the markdown to avoid duplicate work.

Something like this would work:
```
<TestAction acton="runCommand">
```sh
npx create-react-app frontend --template typescript
```
</TestAction>
```